frontend/src/components/marquee/index.tsx

```diff
@@ -25,8 +26,11 @@ const animation = keyframes`
 // Where el must be the marquee text
 const WIDTH_OF_0_FOR_MARQUEE_SIZE = 22.399993896484375;
 
-export const Marquee = ({ message }: MarqueeProps) => {
-  const messageWithSeparator = `${message}`;
+export const Marquee = ({ message, separator = "/" }: MarqueeProps) => {
+  const countSeparator = message.length > 13;
+  const messageWithSeparator = countSeparator
+    ? `${message} ${separator}`
+    : `${message}`;
   const ch = `${messageWithSeparator.length}ch`;
```

Comment:
WIP improvements to this

Using `ch` is def the better solution, what is surprising to me is how often this breaks. Clearly I am missing something on how the ch/textShadow works since it randomly breaks with some texts